qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/AzureBackupAcceptanceIT.java

```diff
@@ -0,0 +1,108 @@
+ *
+ * <p>The tests run against a cluster of 2 brokers and 1 gateway, no embedded gateways, two
+ * partitions and replication factor of 1. This allows us to test that requests are correctly fanned
+ * out across the gateway, since each broker is guaranteed to be leader of a partition.
+ *
+ * <p>NOTE: this does not test the consistency of backups, nor that partition leaders correctly
+ * maintain consistency via checkpoint records. Other test suites should be set up for this.
+ */
+@Testcontainers
+@ZeebeIntegration
+final class AzureBackupAcceptanceIT implements BackupAcceptance {
+  @Container private static final AzuriteContainer AZURITE_CONTAINER = new AzuriteContainer();
+  private static final String CONTAINER_NAME = RandomStringUtils.randomAlphabetic(10).toLowerCase();
+
+  @RegisterExtension
+  @SuppressWarnings("unused")
+  final ContainerLogsDumper logsWatcher =
+      new ContainerLogsDumper(() -> Map.of("azurite", AZURITE_CONTAINER));
+
+  // cannot auto start, as we need azurite to be started before we can configure the brokers
+  @TestZeebe(autoStart = false)
+  private final TestCluster cluster =
+      TestCluster.builder()
+          .withBrokersCount(2)
+          .withGatewaysCount(1)
+          .withReplicationFactor(1)
+          .withPartitionsCount(2)
+          .withEmbeddedGateway(false)
+          .withNodeConfig(this::configureNode)
+          .build();
+
+  private AzureBackupStore store;
+
+  @BeforeEach
+  void beforeEach() {
+    final AzureBackupConfig config =
+        new AzureBackupConfig.Builder()
+            .withConnectionString(AZURITE_CONTAINER.getConnectString())
+            .withContainerName(CONTAINER_NAME)
+            .build();
+    store = new AzureBackupStore(config);
+    final BlobServiceClient blobServiceClient = AzureBackupStore.buildClient(config);
+    blobServiceClient.createBlobContainerIfNotExists(CONTAINER_NAME);
+
+    // we have to configure the cluster here, after azurite is started, as otherwise we won't have
+    // access to the exposed port
+    cluster.brokers().values().forEach(this::configureBroker);
+    cluster.start().awaitCompleteTopology();
+  }
+
+  @AfterEach
+  void afterEach() {
+    CloseHelper.quietCloseAll(() -> store.closeAsync().join());
+  }
+
+  @Override
+  public TestCluster getTestCluster() {
+    return cluster;
+  }
+
+  private void configureBroker(final TestStandaloneBroker broker) {
+    broker.withBrokerConfig(
+        cfg -> {
+          final var backup = cfg.getData().getBackup();
+          final var azure = backup.getAzure();
+
+          backup.setStore(BackupStoreType.AZURE);
+          azure.setBasePath(CONTAINER_NAME);
+          azure.setConnectionString(AZURITE_CONTAINER.getConnectString());
+        });
+  }
+
+  private void configureNode(final TestApplication<?> node) {
+    node.withProperty("management.endpoints.web.exposure.include", "*");
```

Comment:
:wrench: I _think_ this isn't necessary anymore, could you give it one test run without this?